chore: document sdk helpers (#774)
defenseunicorns/pepr: docs/030_user-guide/030_actions/020_validate.md

# Validate

> After the Mutation phase, after all object modifications are complete, and after the incoming object is validated by the API server, validating admission webhooks are invoked and can reject requests to enforce custom policies.

Reviewer: ```suggestion
After the Mutation phase comes the Validation phase where the validating admission webhooks are invoked and can reject requests to enforce custom policies.
```

This is confusing. little wordy too, lots of after. I don't know that the API Server validates before the Validation phase either. I would say to keep it simple.
